src/adapters/postgres/fields-adapter.ts

```
@@ -1044,17 +1050,14 @@ export class PostgresFieldsService implements IServicelocatorfields {
       );
 
       return response;
     } catch (e) {
-      LoggerUtil.error(
-        `${API_RESPONSES.SERVER_ERROR}`,
-        `Error: ${e.message}`,
-      )
+      LoggerUtil.error(`${API_RESPONSES.SERVER_ERROR}`, `Error: ${e.message}`);
       return new ErrorResponse({
-        errorCode: "400",
+        errorCode: '400',
```

ESLint

[error] Replace `'400'` with `"400"`

(prettier/prettier)